scripts/pre_push_hook.py

```diff
@@ -295,6 +295,19 @@ def _install_hook():
         print 'Copied file to .git/hooks directory'
 
 
+def run_frontend_tests_status(files_to_lint):
```

Comment:
This is not clear: it sounds like the function is supposed to run the tests, but instead it's just doing a check. So maybe call it does_diff_include_js_files() instead.

Reply:
Done